flatpak/web/org.nanuc.Axolotl.yml

```diff
@@ -1,12 +1,14 @@
 app-id: org.nanuc.Axolotl
 branch: main
-runtime: org.freedesktop.Platform
-runtime-version: '20.08'
-sdk: org.freedesktop.Sdk
+runtime: org.kde.Platform
```

Comment:
I would expect no changes to flatpak/**web**/org.nanuc.Axolotl.yml but instead a whole new file, at flatpak/**qt**/org.nanuc.Axolotl.yml. Do you see it the same way?

Reply:
Good catch. I had checked out the original version, but this meant my original version, not the current main. This has been fixed in the latest version of this PR.

Reply:
Beautiful!